app/controllers/relationships_controller.rb

```diff
@@ -0,0 +1,32 @@
+class RelationshipsController < ApplicationController
+  before_action :authenticate_user!
+  
+  #  POST "/relationships"
+  def create
```

Comment:
these methods can be moved to users controller with names `follow` and `unfollow`. hence this controller will be removed.